### DISCUSSION TITLE
Is there a way to display note:-entries too in CLion ?

### DISCUSSION THREAD

**Discussion by volkergraf** [johnthagen/clion-cppcheck#99] [General]
Thanks for the great Plougin for CLion. Is there a way to Display node:-Entries that were created by cppcheck ? 

I wrote a little (crappy) piece of C-Code to test cppcheck like

> char *mallocDemo3(int siz){
>....char *foo = (char *)malloc(sizeof(char)*siz);
>....foo[0] = 'X';
>....foo[1] = '\0';
>....return foo;
> }
> ...
> char *pFoo2 = mallocDemo3(-11);

the cppcheck **console** output shows me **notes**, telling me 

>main.c:75:32: note: Calling function 'mallocDemo3', 1st argument '-11' value is -11
>   char *pFoo2 = mallocDemo3(-11);

but i'm not seeing this in CLion ... did I miss something ? The warnings errors etc. are being shown.

thanks and greetings from Germany

Volker

**Comment by firewave** [comment]
The notes are being displayed as a tooltip because the relation would otherwise be lost in the list (as seen with compiler warnings which have notes). There are `...` at the end of the message if there is a tooltip available. See #75.

**Comment by volkergraf** [comment]
Ahh now I see, thank you
